src/core/server/services/comments/pipeline/pipeline.ts

```diff
@@ -44,8 +48,15 @@ export interface ModerationPhaseContext {
   req?: Request;
 }
 
+export interface ModerationPhaseContext extends ModerationPhaseContextInput {
+  /**
+   * stripped is the HTML stripped version of the comment body.
+   */
+  strippedTags: string;
```

Comment:
I find this naming a bit confusing, it suggests that it stores the tags that were stripped, not the remaining string. Maybe `strippedString` or `htmlStripped` or `tagStripped`?